scripts/join-release-testnet.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# Set up a Exocore service to join the Exocore Release testnet.
+
+# Load environment variables from .env file
+if [ -f .env ]; then
+  export $(cat .env | xargs)
+fi
+
+# Configuration
+# You can have these values set in .env file, and run 'source .env' before running this script.
+# ***
+NODE_HOME=${NODE_HOME:-$HOME/.exocored}
+NODE_MONIKER=${NODE_MONIKER:-release-testnet}
+STATE_SYNC=${STATE_SYNC:-true}
+STATE_SYNC=$(echo "$STATE_SYNC" | tr '[:upper:]' '[:lower:]')
+
+if [ "$STATE_SYNC" = "true" ]; then
+  STATE_SYNC=true
+else
+  STATE_SYNC=false
+fi
+
+# Exocore configuration
+# Default values for Exocore testnet
+SERVICE_NAME=exocore
+EXOCORE_VERSION=1.0.2
+CHAIN_BINARY_URL=https://github.com/ExocoreNetwork/exocore/releases/download/v$EXOCORE_VERSION/exocore_$EXOCORE_VERSION\_Linux_amd64.tar.gz
+GAS_PRICE=0.0001aexo
+# ***
+
+CHAIN_BINARY='exocored'
+CHAIN_ID=exocoretestnet_233-4
+GENESIS_ZIPPED_URL=https://github.com/ExocoreNetwork/testnets/raw/main/genesis/exocoretestnet_233-4.json
+SEEDS="5dfa2ddc4ce3535ef98470ffe108e6e12edd1955@seed2t.exocore-restaking.com:26656,4cc9c970fe52be4568942693ecfc2ee2cdb63d44@seed1t.exocore-restaking.com:26656"
+SYNC_RPC_1=http://seed1t.exocore-restaking.com:26657
+SYNC_RPC_2=http://seed2t.exocore-restaking.com:26657
+SYNC_RPC_SERVERS="$SYNC_RPC_1,$SYNC_RPC_2"
+
+# Install wget and jq
+sudo apt-get install curl jq wget -y
```

Comment:
It's generally better to avoid `sudo` within scripts because it leads to inconsistent privilege levels that are hard to debug. Most scripts that I have seen in the wild, require the user to run it via as the root user (and check this in the beginning). However, this is not a blocker to merge.

Reply:
`sudo` still here; do you think we should retain it? I saw the README comment `Script must be run either as root or from a sudoer account.`

Reply:
yes, and it is very obvious that only curl,jq,wget will be installed.